lib/widgets/message_list.dart

```diff
@@ -852,7 +852,7 @@ class _MarkAsReadWidgetState extends State<MarkAsReadWidget> {
                 backgroundColor: WidgetStatePropertyAll(messageListTheme.unreadMarker),
               ),
               onPressed: _loading ? null : () => _handlePress(context),
-              icon: const Icon(Icons.playlist_add_check),
+              icon: const Icon(ZulipIcons.message_checked),
```

Comment:
Ah: I missed the relevant discussion and ended up opening an issue for this change 🙂 please add `Fixes #1357.` to the commit message and PR description. Thanks!